Assets/PatchKit Patcher/Scripts/InputArgumentsPatcherDataReader.cs

```diff
@@ -63,9 +63,15 @@ public PatcherData Read()
             {
                 throw new ApplicationException("Unable to parse app data path from command line.");
             }
-
             data.AppDataPath = MakeAppDataPathAbsolute(relativeAppDataPath);
 
+            string lockFilePath;
+            if (!TryReadArgument("--lockfile", out lockFilePath))
+            {
+                throw new ApplicationException("Unable to parse lock file path from command line.");
```

Comment:
--lockfile is something optional for now. We shouldn't stop the patcher if it's not passed in. Please also log in that it's being used. Also log a warning if it cannot be found.